prow/hook/events.go

```diff
@@ -339,7 +339,7 @@ func (s *Server) handleGenericComment(l *logrus.Entry, ce *github.GenericComment
 				ce.Repo.Name,
 				ce.Number,
 			)
-			if err := h(pc, *ce); err != nil {
+			if err := h(pc, *ce, s.Metrics.WebhookCounter); err != nil {
```

Comment:
s/WebhookCounter/PluginsCounter/
I think the `PluginsCounter` or `Metrics` struct should be provided to the handler as a field in the plugin client instead of as an additional parameter.

Reply:
ack, good point